addon/components/ilios-calendar-event-month.js

```diff
@@ -38,6 +38,18 @@ export default CalendarEvent.extend({
   isIlm: notEmpty('event.ilmSession'),
   isOffering: notEmpty('event.offering'),
   clickable: any('isIlm', 'isOffering'),
+
+  recentlyUpdated: computed('event', {
+    get() {
+      const event = this.event;
```

Comment:
It's currently undocumented and I talked it over with my friend. Going to use the Ember getter.